### PR TITLE
view title photos and collections

### PR DESCRIPTION
**Pull request by VuManhHung97**
Handle swiper TabLayout ,ViewPager
Create UI : https://marvelapp.com/c1cbha4/screen/58854270

https://edu-redmine.sun-asterisk.vn/issues/14512

### CODE REVIEW

**Comment by VuManhHung97** [comment]
ready

File: app/src/main/java/com/sun_asterisk/myeditor03/screen/collections/CollectionsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.sun_asterisk.myeditor03.R
+
+class CollectionsFragment : Fragment() {
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/myeditor03/screen/collections/CollectionsFragment.kt:12]: cách ra 1 dòng giữa class và method cho dễ nhìn code e

File: app/src/main/java/com/sun_asterisk/myeditor03/screen/collections/CollectionsFragment.kt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        val view = inflater.inflate(R.layout.fragment_collections, container, false)
+        return view
+    }
+
+    companion object {
+        fun getInstance(): CollectionsFragment {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/myeditor03/screen/collections/CollectionsFragment.kt]:
```suggestion
        fun instance(): CollectionsFragment {
```

File: app/src/main/java/com/sun_asterisk/myeditor03/screen/home/adapter/PagerAdapter.kt [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package com.sun_asterisk.myeditor03.screen.home.adapter
+
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentPagerAdapter
+import java.util.ArrayList
+
+class PagerAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {
+    private val fragmentList = ArrayList<Fragment>()
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/myeditor03/screen/home/adapter/PagerAdapter.kt]:
```suggestion
    private val fragments = ArrayList<Fragment>()
```

File: app/src/main/java/com/sun_asterisk/myeditor03/screen/home/adapter/PagerAdapter.kt [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    private val fragmentTitleList = ArrayList<String>()
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/myeditor03/screen/home/adapter/PagerAdapter.kt]:
```suggestion
    private val fragmentsTitle = ArrayList<String>()
```

File: app/src/main/java/com/sun_asterisk/myeditor03/screen/photos/PhotosFragment.kt [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.sun_asterisk.myeditor03.R
+
+class PhotosFragment : Fragment() {
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/myeditor03/screen/photos/PhotosFragment.kt:12]: as above

File: app/src/main/java/com/sun_asterisk/myeditor03/screen/photos/PhotosFragment.kt [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+        val view = inflater.inflate(R.layout.fragment_photos, container, false)
+        return view
+    }
+
+    companion object {
+        fun getInstance(): PhotosFragment {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/myeditor03/screen/photos/PhotosFragment.kt]: as above

File: app/src/main/java/com/sun_asterisk/myeditor03/utils/CommonUtils.kt [hunk 7]
```diff
@@ -1,6 +1,6 @@
 package com.sun_asterisk.myeditor03.utils
 
-class CommomUtils {
+class CommonUtils {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/myeditor03/utils/CommonUtils.kt]:
```suggestion
object CommonUtils {
```

File: app/src/main/java/com/sun_asterisk/myeditor03/utils/CommonUtils.kt [hunk 8]
```diff
@@ -1,6 +1,6 @@
...
     interface Genres {
         companion object {
             const val PHOTO = "Photos"
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/myeditor03/utils/CommonUtils.kt]:
```suggestion
interface Genres {
       const val PHOTO = "Photos"
}
```

**Comment by VuManhHung97** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**